base_wamas_ubl/tests/samples/UBL2WAMAS-SAMPLE_KRETK_KRETP.wamas

```diff
@@ -1,6 +1,6 @@
-ODOO      WAMAS     00000120230501020000KRETK0051000130377              HOST                     STDMAN0001040         MyCompany                                                                                                                                                                      1839 Arbor Way                          California                              95380     Turlock                                                                         US  (623)-853-7197                                                        wood.corner26@example.com                                                                                                                                                          20230427000000          
```

Comment:
AdditionalStreetName_1 doesn't appear here?

Reply:
> AdditionalStreetName_1 doesn't appear here?

I haven't added it to the base_wamas_ubl/tests/samples/UBL2WAMAS-SAMPLE_KRETK_KRETP.xml file yet so it's not there yet. I will add it.